src/App.vue

```diff
@@ -1053,19 +1080,19 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
   }
 
   /** Updates Launch Darkly with current user info. */
-  private async updateLaunchDarkly (userInfo: any): Promise<void> {
+  private async updateLaunchDarkly (userInfo: any, keycloakRoles: string[]): Promise<void> {
     // since username is unique, use it as the user key
     const key: string = userInfo.username
     const email: string = userInfo.contacts[0]?.email || userInfo.email
     const firstName: string = userInfo.firstname
     const lastName: string = userInfo.lastname
-    // remove leading { and trailing } and tokenize string
-    const custom: any = { roles: userInfo.roles?.slice(1, -1).split(',') }
```

Comment:
Out of curiosity, does this syntax work? So:
```javascript
const custom: any = { roles: keycloakRoles }
```

Reply:
Yes, they're equivalent. Pick which style you prefer.

I _slightly_ prefer the `x = something as type` syntax because the type isn't always needed and I like to see a lot of `x =` (for consistency).

Reply:
Alright, got it 👍 Thanks!